scripts/condor/spawn_condor_clients.py

```diff
@@ -49,23 +46,13 @@ def make_condor_file(  # pylint: disable=R0913,R0914
             else ""
         )
 
-        transfer_input_files: List[str] = []
-        executable = os.path.abspath("./scripts/launch_scripts/docker/launch_client.sh")
+        transfer_input_files: List[str] = [startup_json]
 
         # write
-        args = (
-            f"--mq-basename {mq_basename} "
-            f"--gcd-dir {gcd_dir} "
-            f"--broker {broker} "
-            f"--auth-token {auth_token} "
-            f"--log {log} "
-            f"--log-third-party {log_third_party} "
-            f"--timeout-to-clients {timeout_to_clients} "
-            f"--timeout-from-clients {timeout_from_clients}"
-        )
         file.write(
-            f"""executable = {executable}
-arguments = {args}
+            f"""executable = python -m skymap_scanner.client
```

Comment:
The `executable` can be only one thing, so `python` in this case.  You'd need to add the rest to `arguments`.

Reply:
Okay :+1: